definitions/npm/react-redux_v4.x.x/flow_>=v0.24.x/react-redux_v4.x.x.js

```diff
@@ -0,0 +1,33 @@
+type ReactReduxConnectAll = <D, P, S, C: React$Component<D, P, S>, SP, DP, Dispatch: Function>(
+  mapStateToProps: (state: Object) => SP,
+  mapDispatchToProps: (dispatch: Dispatch) => DP,
```

Comment:
`mapDispatchToProps` can also accept an object, which it doesn't look like this definition covers? https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments